### ISSUE TITLE
It's too easy to delete a space

### ISSUE THREAD

**Issue by marcoambrosini** [anyproto/anytype-ts#946]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing requests and didn't find any that were similar
- [X] I have considered creating a pull request instead and want to proceed

### Clear and concise description of the problem

It's too easy to delete a space and this action cannot be undone. Today I deleted a space with a similar icon to an empty one and lost quite a bit of data.

### Suggested solution

Have me type the name of the space in an input field to delete 

### Alternative

Allow me to recover a deleted space

**Comment by e1sordo** [comment]
I totally agree! It's such a simple task but so rewarding!

**Comment by muava12** [comment]
yes i also feel the same..  may be promt to manulally typing space name before deleting can make for confirmation..

**Comment by lanesawyer** [comment]
I believe this has been solved for the desktop version and can be closed? Or is there more work to be done?

**Comment by marcoambrosini** [comment]
No, It's still relevant, there's a simple confirmation dialog only still

**Comment by ghost** [comment]
For me fixed in 0.50.1 (pre-release) at Windows

@marcoambrosini 
<img width="873" height="772" alt="Image" src="https://github.com/user-attachments/assets/3b6e882a-850f-4bad-99c9-fd34dc1d396c" />

**Comment by marcoambrosini** [comment]
Thanks <3
